packages/components/app-icon/src/app-icon.js

```diff
@@ -10,21 +10,20 @@ customElementDefineHelper(
 
 		static get properties() {
 			return {
-				// Use the type when have more sizes of app icon
-				type: { type: String, reflect: true },
+				size: { type: String, reflect: true },
 				src: { type: String, reflect: true },
 				alt: { type: String, reflect: true }
 			};
 		}
 
 		constructor() {
 			super();
-			this.type = '';
+			this.size = '';
 		}
 
 		render() {
 			return html`
-				<img .src="${this.src}" .alt="${this.alt}"></img>
+				<img .src="${this.src}" .alt="${this.alt}" size="${this.size}"></img>
```

Comment:
Since you used the `:host([size='large'])` this size attribute and constructor are redundant. You should either remove them or change the css selector 👍